### PR TITLE
Add task

### PR DESCRIPTION
**Pull request by ananya21**
AddTaskCommand, AddTaskParser, e.t.c Fixed checkstyle violations

### CODE REVIEW

File: src/main/java/seedu/address/model/project/Task.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    private final Name taskName;
+
+    private Person person;
+
+    private boolean status;
+    private Integer progressCounter = 0;
+
+    private LocalDate deadlineDate;
+
+    private Integer priorityNumber;
```
Comment (Lin-Shuang-Shuang) [inline src/main/java/seedu/address/model/project/Task.java:22]: Instead of this perhaps an "urgent" status or "important" status would be better?

**Review by Lin-Shuang-Shuang — APPROVED** [review]
LGTM! The tests can be added in another PR.

File: src/main/java/seedu/address/logic/parser/AddProjectCommandParser.java [hunk 2]
```diff
@@ -26,7 +29,8 @@ public AddProjectCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddProjectCommand.MESSAGE_USAGE));
         }
         Name name = ParserUtil.parseName(projectName);
-        Person person = new Person(name);
+        List<Task> taskList = new ArrayList<>();
+        Person person = new Person(name, taskList);
```
Comment (modembcc) [inline src/main/java/seedu/address/logic/parser/AddProjectCommandParser.java:33]: Is it better to just use the old version if it works the same way?

File: src/main/java/seedu/address/model/project/Task.java (hunk 1 above)
Comment (modembcc) [reply src/main/java/seedu/address/model/project/Task.java:22]: I think priorityNumber and progressCounter shouldn't be implemented before we get the MVP, but I think we can fix that in the next pr as well.

**Review by modembcc — APPROVED** [review]
LGTM, can merge.

**Pull request merged**